packages/elements/src/components/ino-tooltip/ino-tooltip.tsx

```diff
@@ -138,6 +138,7 @@ export class Tooltip implements ComponentInterface {
       delay: this.delay,
       placement: this.placement,
       trigger: this.trigger,
+      arrow: true,
```

Comment:
As i don't know the proper design requirements i'm not sure if we should expose this option to developers or if we should hardcode it. In generel it will also improve developer experience for consumers if we are consistent in API design of both ino-tooltip & ino-popover

Reply:
This change is already implemented In PR #975